librz/core/cmd/cmd_eval.c

```diff
@@ -53,8 +53,41 @@ static bool pal_seek(RzCore *core, RzConsPalSeekMode mode, const char *file, RzL
 	return true;
 }
 
-RZ_API bool rz_core_theme_load(RzCore *core, const char *name) {
+static bool print_meta_info(RzCore *core, RzCmdStateOutput *state, RzThemeInfo *MetaInfo, const char *name) {
```

Comment:
```suggestion
static bool print_meta_info(RzCore *core, RzCmdStateOutput *state, RzThemeInfo *meta_info, const char *name) {
```

We do not use such names usually, replace with `meta_info`

Reply:
Also, I'd say that `name` should also be part of `RzThemeInfo`